seimas/settings/development.py

```diff
@@ -3,3 +3,10 @@
 from seimas.settings.base import *  # noqa
 
 DEBUG = True
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': 'seimas.db',
+    }
+}
```

Comment:
It is better to setup PostgreSQL, because SQLite and PostgreSQL have quite many differences. So remove this and use database settings defined in `seimas/settings/base.py`.